deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -982,3 +982,14 @@ Example:
 {{- define "kubernetes.minor" -}}
 {{- print (regexFind "^\\d+" .Capabilities.KubeVersion.Minor) -}}
 {{- end -}}
+
+{{- define "fluentd.metadata.annotations_match.quotes" -}}
+{{- $matches_with_quotes := list -}}
+{{- range $match := .Values.fluentd.metadata.annotation_match  }}
+{{- $match_with_quotes := printf "\"%s\"" $match }}
```

Comment:
please use `quote` function as someone potentially can put quotes in variable

Reply:
https://github.com/Masterminds/sprig/blob/master/docs/strings.md#quote-and-squote

Reply:
I've tested with `quote` and I noticed that it adds `\` before special characters. I used this in following way:
```
{{- $match_with_quotes := $match  | quote }}
```
and then in help templates there was following form:
```
annotation_match ["sumologic\\.com.*","tailing-sidecar.sumologic\\.com.*","test-sumologic\\.com.*","test-tailing-sidecar.sumologic\\.com.*"]
```
and in fluend logs I saw that Pod had following configuration
```
 <filter containers.**>
      @type kubernetes_metadata
      @log_level "error"
      annotation_match ["sumologic\\.com.*","tailing-sidecar.sumologic\\.com.*","test-sumologic\\.com.*","test-tailing-sidecar.sumologic\\.com.*"]
      de_dot false
      watch true
      ca_file ""
      verify_ssl true
      client_cert ""
      client_key ""
      bearer_token_file ""
      cache_size 10000
      cache_ttl 7200
      tag_to_kubernetes_name_regexp ".+?\\.containers\\.(?<pod_name>[^_]+)_(?<namespace>[^_]+)_(?<container_name>.+)-(?<docker_id>[a-z0-9]{64})\\.log$"
    </filter>
```